test/integration_tests/test_models.py

```diff
@@ -0,0 +1,67 @@
+import torch
+import torchtext
+
+from ..common.assets import get_asset_path
+from ..common.torchtext_test_case import TorchtextTestCase
+
+
+class TestModels(TorchtextTestCase):
+    def test_xlmr_base(self):
```

Comment:
It seems to me that the underlying logics are same.
You might want to consider parameterizing the tests.